### PR TITLE
Bower improvements

### PR DESCRIPTION
**Pull request by brousalis**
Adds a dist folder for bower
Using gulp for the build process
Fixed an issue with the dropdown and $apply collision

### CODE REVIEW

File: vendor/assets/stylesheets/rolodex/settings/mixins/_svg.sass [hunk 1]
```diff
@@ -3,6 +3,8 @@
 
 =background-svg($file-name, $size...)
```
Comment (DLavin23) [inline vendor/assets/stylesheets/rolodex/settings/mixins/_svg.sass]: :+1:

**Comment by DLavin23** [comment]
lgtm :game_die:

File: gulpfile.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+  return gulp.src(path.join(assetPath, '/**/*.coffee'))
+    .pipe($.coffee())
+    .pipe($.ngAnnotate())
+    .pipe(gulp.dest(tmpPath))
+});
+
+// A hack for non-Ruby projects to fake the templates namespace (which lives in a gem)
+gulp.task('js-hack', function() {
+  var stream = source('templates.js');
+  stream.end("(function() { angular.module('templates', []); }).call(this);");
```
Comment (ajself) [inline gulpfile.js]: I wouldn't encourage calling this a hack as a name for a task. How about `js-template-cache` or some such

File: gulpfile.js [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+    .pipe(gulp.dest(tmpPath));
+});
+
+// Builds the dist files
+gulp.task('build', ['js', 'templates'], function() {
+  return gulp.src(path.join(tmpPath, '/**/*.js'))
+    .pipe($.concat('rolodex.js'))
+    .pipe(gulp.dest(distPath))
+    .pipe($.uglify())
+    .pipe($.rename({extname: ".min.js"}))
```
Comment (ajself) [inline gulpfile.js]: Nix these dub quotes

**Comment by ajself** [comment]
Do these template files have to end in `.ngt` any more?

**Comment by brousalis** [comment]
Going to keep them as `.ngt` instead of `.html` to still support the angular-html2js gem.

**Pull request merged**